### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#439] [Esami & Esoneri M-Z]
## Traccia

<img width="636" alt="image" src="https://github.com/user-attachments/assets/5addb2c3-47ff-4561-9cf0-f1e663f74c5d">

**Comment by rimaout** [comment]
![image](https://github.com/user-attachments/assets/705966a1-517f-4dd0-b0d4-c071e17d683a)

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/9b1ddfe1-fda9-4b4c-9fd6-eb03ec1f32da)
